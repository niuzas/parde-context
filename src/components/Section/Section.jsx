import React from 'react';
import PropTypes from 'prop-types';

const TextInput = ({ children }) => (
  <section className="section">
    <div className="container">{children}</div>
  </section>
);

TextInput.propTypes = {
  children: PropTypes.node.isRequired
};

export default TextInput;
