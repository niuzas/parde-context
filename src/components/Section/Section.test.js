import React from 'react';
import renderer from 'react-test-renderer';

import Section from './Section';

it('Section renders correctly', () => {
  const tree = renderer.create(<Section>Content</Section>).toJSON();
  expect(tree).toMatchSnapshot();
});
