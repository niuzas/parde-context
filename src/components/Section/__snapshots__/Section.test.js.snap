// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
<section
  className="section"
>
  <div
    className="container"
  >
    Content
  </div>
</section>
`;
